code/game/shuttle_engines.dm

```diff
@@ -6,10 +6,6 @@
 	armor = list(melee = 100, bullet = 10, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 70) //default + ignores melee
 
 /obj/structure/shuttle/shuttleRotate(rotation)
```

Comment:
Put a return inside this empty proc instead of just a blank override, maybe even with a comment